samples/net/mqtt_publisher/README.rst

```diff
@@ -114,6 +114,27 @@ Open another terminal window and type:
 
 	$ mosquitto_sub -t sensors
 
+Building/Running On Linux
+==========================
+
+This sample can also be built and run on Linux.  The setup of the mosquitto
```

Comment:
For clarity, I'd add ``natively`` to the phrase: ``built and run natively on Linux``

Reply:
Done: thanks for the review.